wazuh-indexer/Dockerfile

```diff
@@ -0,0 +1,72 @@
+# Wazuh Docker Copyright (C) 2021 Wazuh Inc. (License GPLv2)
+FROM ubuntu:focal AS builder
+
+RUN apt-get update -y && apt-get install curl openssl xz-utils -y
+
+COPY config/unattended_installer.tar.gz /
+
+COPY config/config.sh .
+
+COPY config/config.yml /
+
+RUN tar -xzvf /unattended_installer.tar.gz
+
+RUN mv /config.yml /unattended_installer/install_functions/
+
+RUN chmod 755 /unattended_installer/install_functions/wazuh-cert-tool.sh && bash /unattended_installer/install_functions/wazuh-cert-tool.sh
```

Comment:
The wazuh-cert-tool.sh must be downloaded as explained.

Reply:
Done